content/en/monitors/guide/how-to-set-up-rbac-for-monitors.md

```diff
@@ -0,0 +1,216 @@
+
+### UI
+
+All new monitors created from the UI use the new `restricted_roles` parameter. 
+All monitors (existing and new) also display the new role restriction option regardless of the mechanism used under the hood:
+
+{{< img src="/monitors/guide/monitor_rbac_non_restricted.jpg" alt="RBAC Non Restricted Monitor"  >}}
+
+Existing monitors have their definitions updated from the old locked mechanism to the new role restriction one on the fly whenever a monitor is saved.
+
+Below are some instructions to help you determine how to proceed in case you need to save a monitor that is using the previous locked mechanism.
+
+#### Locked monitors (`locked:true`) edited by creator or user with Datadog Admin Role
+
+You are a user with [Datadog Admin Role][2] and/or are the creator of the monitor. You are editing a locked monitor and seeing the below warning:
+
+<TO DO: ADD SCREENSHOT OF WARNING>
+
+On save, your monitor's definition will be updated to all roles with Admin permissions.
+You have different ways to handle this warning depending on the changes you are willing to make on your monitor:
+
+**1. You do not want to change anything about your monitor permissions**
+
+Save the monitor to have the update from the locked mechanism to the restricted_roles one be performed automatically, along with any other updates you might have done on your monitor (e.g. threshold update, message, etc.).
+
+If you were just opening the edit page to see more details about your monitor configuration, you can also hit **Save**. That also performs the update for your monitor.
+
+**2. You want to allow all users to edit this monitor**
+
+Save the monitor, edit it again, and remove all roles that were generated for this monitor. 
+
+**3. You want your monitor to be restricted to some roles, but not the exact equivalent of the previous locked option**
+
+Select the roles you want this monitor to be restricted to. This will override the locked equivalent set of roles to only the ones you specify.
+  
+#### Locked monitors (`locked:true`) edited by non creator or user without Datadog Admin Role
+
+You are a user without [Datadog Admin Role][2] and/or are not the creator of the monitor. You are editing a locked monitor and seeing the below warning:
+
+<TO DO: ADD SCREENSHOT OF WARNING>
+
+As mentioned in the warning, this monitor was locked in the previous system. You should consequently reach out to a user with a [Datadog Admin Role][2] or to the creator of the monitor for them to update the monitor to include one of the roles you belong to. Your admin will have to follow guidelines in the [above section](#locked-monitors-lockedtrue-edited-by-creator-or-user-with-datadog-admin-role) (2. or 3.).
+
+**Note:** The discrepancy you are seeing between the warning and the option is expected:
+
+<TO DO: ADD SCREENSHOT OF WARNING AND OPTION>
```

Comment:
Replace the TO DO with the text of the warning.

Reply:
Completely got rid of that one, I don't think it's necessary to add the warning as it's mentioned above.